app/views/case_workers/claims/_claims.html.haml

```diff
@@ -28,14 +28,14 @@
           %td
             = link_to claim.case_number , case_workers_claim_path(claim),{:class => 'js-test-case-number-link'}
           %td
-            = claim.advocate.user.name
+            = claim.advocate_name
           %td
             = claim.defendant_names
           %td.js-test-total
             = claim.total_inc_vat
           - if params[:action] == 'archived'
             %td
-              = claim.state.humanize
+              = claim.humanized_state
```

Comment:
not sure this presenter refactor is worth while since humanize, is pretty ubiquitous as a kind of presenter method in itself - so its kind of a presenter using a presenter. It also adds a layer of abstraction without improving clarity.